cosmic/resource_cosmic_vpc.go

```diff
@@ -15,6 +15,9 @@ func resourceCosmicVPC() *schema.Resource {
 		Read:   resourceCosmicVPCRead,
 		Update: resourceCosmicVPCUpdate,
 		Delete: resourceCosmicVPCDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCosmicVPCInstanceImport,
```

Comment:
You can just use this instead: `State: schema.ImportStatePassthrough` and then drop the custom `resourceCosmicVPCInstanceImport` function 👍